volksbot:
  ros__parameters:
    wheel_radius: 0.13
    axis_length: 0.65
    drive_backwards: false
    turning_adaptation: 0.99
    num_wheels: 4
    publish_tf: false

